modules/multiqc/main.nf

```diff
@@ -20,8 +21,13 @@ process MULTIQC {
 
     script:
     def args = task.ext.args ?: ''
+    def config = multiqc_config ?: "--config $multiqc_config"
```

Comment:
```suggestion
    def config = multiqc_config ? "--config $multiqc_config" : ''
```
Only want the config when the value is set, not unset.

Reply:
Oh right, stupid mistake